cirq/aqt/aqt_sampler.py

```diff
@@ -0,0 +1,224 @@
+        """
+        while True:
+            time.sleep(1.0)
+            data = put(self.remote_host,
+                       data={
+                           'data': json_str,
+                           'id': id_str,
+                           'acccess_token': self.access_token,
+                           'repetitions': repetitions,
+                           'num_qubits': num_qubits
+                       }).json()
+            if data['status'] == 'finished':
+                break
+        measurements_int = data['samples']
+        measurements = np.zeros((len(measurements_int), num_qubits))
+        for i, result_int in enumerate(measurements_int):
+            for j in range(num_qubits):
+                measurements[i, j] = np.floor(result_int / 2**j)
+        return measurements
+
+    def run_sweep(self,
+                  program: Union[circuits.Circuit, schedules.Schedule],
+                  params: study.Sweepable,
+                  repetitions: int = 1,
+                  num_qubits: int = 1) -> List[study.TrialResult]:
+        """Samples from the given Circuit or Schedule.
+
+        In contrast to run, this allows for sweeping over different parameter
+        values.
+
+        Args:
+            program: The circuit or schedule to simulate.
+            Should be generated using AQTSampler.generate_circuit_from_list
+            params: Parameters to run with the program.
+            repetitions: The number of repetitions to simulate.
+            num_qubits: The number of qubits in the system.
+
+        The parameters remote_host and access_token are not used.
+
+        Returns:
+            TrialResult list for this run; one for each possible parameter
+            resolver.
+        """
+        # TODO: Where should we get the num_qubits??
+        # TODO: Probably from the measurement in the circuit!
+        meas_name = 'm'  # TODO: Get measurement name from circuit
+        circuit = (program if isinstance(program, circuits.Circuit) else
+                   program.to_circuit())
+        param_resolvers = study.to_resolvers(params)
+        trial_results = []  # type: List[study.TrialResult]
+        for param_resolver in param_resolvers:
+            id_str = uuid.uuid1()
+            json_list = self._run_api(circuit=circuit,
+                                      param_resolver=param_resolver)
+            results = self._send_json(json_str=json_list,
+                                      id_str=id_str,
+                                      repetitions=repetitions,
+                                      num_qubits=num_qubits)
+            results = results.astype(bool)
+            res_dict = {meas_name: results}
+            trial_results.append(
+                study.TrialResult(params=param_resolver,
+                                  repetitions=repetitions,
+                                  measurements=res_dict))
+        return trial_results
+
+    def run(self,
```

Comment:
The default `num_qubits` here disagrees with the one in run_sweep.

If `num_qubits` is removed from `run_sweep` this method can be deleted.